src/forms/editors/TextFieldEditor.jsx

```diff
@@ -21,28 +21,72 @@ export default class TextFieldEditor extends Component {
     this.setState({ field: updatedField });
   }
 
+  onMinCharsChange(e) {
+    let { field } = this.state;
+    let updatedProps = Object.assign({}, field.props, { minLength: e.target.value });
+    let updatedField = Object.assign({}, field, { props: updatedProps });
+    this.props.onEditorChange(updatedField);
+  }
+
   onMaxCharsChange(e) {
     let { field } = this.state;
     let updatedProps = Object.assign({}, field.props, { maxLength: e.target.value });
     let updatedField = Object.assign({}, field, { props: updatedProps });
     this.props.onEditorChange(updatedField);
   }
 
+  onMinLengthChange(e) {
+    this.setState({ minLengthEnabled: e.target.checked });
+    // Set to 0 if disabled
+    if (!e.target.checked) {
+      let { field } = this.state;
+      let updatedProps = Object.assign({}, field.props, { minLength: 0 });
+      let updatedField = Object.assign({}, field, { props: updatedProps });
+      this.props.onEditorChange(updatedField);
+    }
+  }
+
+  onMaxLengthChange(e) {
+    this.setState({ maxLengthEnabled: e.target.checked });
+    // Set to 0 if disabled
+    if (!e.target.checked) {
+      let { field } = this.state;
+      let updatedProps = Object.assign({}, field.props, { maxLength: 0 });
+      let updatedField = Object.assign({}, field, { props: updatedProps });
+      this.props.onEditorChange(updatedField);
+    }
+  }
+
   render() {
     let { field } = this.state;
     return (
       <div>
+
         <div style={ styles.bottomOptions }>
 
           <div style={ styles.bottomOptionsLeft }>
             <label style={ styles.bottomCheck }>
-                Max. chars:
-                <input
-                  onChange={ this.onMaxCharsChange.bind(this) }
-                  defaultValue={ field.props.maxLength || 0 }
-                  type="text"
-                  style={ styles.bottomCheckTextInput }></input>
+              <input type="checkbox" checked={ this.state.minLengthEnabled } onChange={ this.onMinLengthChange.bind(this) } />
```

Comment:
we have a `Checkbox` component. It's not necessary to always use it, but let's try to use it for style consistency if you remember.